### PR TITLE
Fix failing macvim installation on rake install

### PR DESCRIPTION
**Pull request by hazz1925**
Fix macvim installation failing since homebrew updates to not support options anymore.

### CODE REVIEW

**Comment by hazz1925** [comment]
Need help to test this on mac since I can't test this using docker-compose because its on linux and not macOS.

File: Rakefile [hunk 1]
```diff
@@ -173,7 +173,8 @@ def install_homebrew
   puts "Installing Homebrew packages...There may be some warnings."
   puts "======================================================"
   run %{brew install zsh ctags git hub tmux reattach-to-user-namespace the_silver_searcher ghi}
-  run %{brew install macvim --with-override-system-vim --with-lua --with-luajit}
+  run %{brew tap macvim-dev/macvim}
+  run %{brew install --HEAD macvim-dev/macvim/macvim}
```
Comment (lfilho) [inline Rakefile]: Why do we need `--HEAD` here? Won't that risk installing unstable versions of MacVim?

**Comment by lfilho** [comment]
Hey @hazz1925, thanks for your contribution! Since I'm not actively using this fork anymore nor macvim (i'm using nvim), I'll wait until someone confirm this works before merging. Also, I have left a comment. Please check it out :-)

**Comment by vovinacci** [comment]
@hazz1925
https://github.com/macvim-dev/homebrew-macvim is marked as deprecated in favour of 'native' HomeBrew formula. To keep it simple, my proposal would be to simplify failing command to just 
```
run %{brew install macvim}
```
This works for me locally for some time already.
If no reply for some time here, I'll create PR by myself.

@lfilho Just out my curioucity, what fork do you use? :)

**Comment by lfilho** [comment]
Merged #827 that takes care fo this. Thanks for your contributions.

@vovinacci Yup, you found it! :) Thanks again for the PR

**Pull request closed without merging**